package.json

```diff
@@ -16,72 +16,76 @@
     "example"
   ],
   "license": "AGPL3.0-only",
+  "scripts": {
+    "dev": "vite",
+    "build": "tsc && vite build",
+    "deploy": "gh-pages -d example/build",
+    "commit-msg": "commitlint --edit $1",
+    "hooks:uninstall": "husky uninstall",
+    "hooks:install": "husky install",
+    "lint": "eslint .",
+    "prettier:check": "prettier --check \"{src,test}/**/*.ts\"",
+    "prettier:write": "prettier --write \"{src,test}/**/*.ts\"",
+    "type-check": "tsc --noEmit",
+    "check": "yarn lint && yarn prettier:check && yarn type-check",
+    "pre-commit": "yarn prettier:check && yarn lint",
+    "predeploy": "cd example && yarn install && yarn run build",
+    "prepare": "yarn build",
+    "prepack": "yarn build",
+    "post-commit": "git status",
+    "start:example": "yarn workspace graasp-query-client-example start",
+    "test": "jest --forceExit --coverage --verbose --silent",
+    "test:watch": "yarn test --watchAll",
+    "test:ci": "yarn && cd example && yarn && cd .. && yarn test"
+  },
   "dependencies": {
-    "@graasp/sdk": "github:graasp/graasp-sdk",
+    "@graasp/sdk": "0.11.0",
     "axios": "0.27.2",
     "http-status-codes": "2.2.0",
-    "immutable": "4.1.0",
+    "immutable": "4.3.0",
     "miragejs": "0.1.46",
     "qs": "6.11.0",
-    "react-query": "4.0.0-beta.23",
-    "standard-version": "9.5.0",
-    "tsc": "2.0.4",
     "uuid": "9.0.0"
   },
   "devDependencies": {
     "@commitlint/cli": "17.3.0",
     "@commitlint/config-conventional": "17.3.0",
+    "@tanstack/react-query": "4.28.0",
+    "@tanstack/react-query-devtools": "4.28.0",
     "@testing-library/react-hooks": "8.0.0",
     "@types/jest": "28.1.2",
     "@types/node": "17.0.45",
     "@types/qs": "6.9.7",
-    "@types/react": "18.0.14",
-    "@types/uuid": "8.3.4",
-    "@typescript-eslint/eslint-plugin": "5.28.0",
-    "@typescript-eslint/parser": "5.28.0",
+    "@types/react": "18.0.28",
+    "@types/react-dom": "18.0.11",
+    "@types/uuid": "9.0.1",
+    "@typescript-eslint/eslint-plugin": "5.56.0",
+    "@typescript-eslint/parser": "5.56.0",
+    "@vitejs/plugin-react": "3.1.0",
     "eslint": "8.29.0",
     "eslint-config-prettier": "8.5.0",
     "husky": "8.0.2",
     "jest": "28.1.3",
-    "microbundle": "0.15.1",
     "nock": "13.2.9",
     "prettier": "2.8.1",
+    "react": "18.2.0",
+    "react-dom": "18.2.0",
```

Comment:
Same as above

Reply:
It is still needed that you have the packages when running i.e. the tests, so this is why they are `devdependencies`